src/test/java/com/google/sps/data/ScheduledInterviewServletTest.java

```diff
@@ -54,143 +54,218 @@
   LocalServiceTestHelper helper =
       new LocalServiceTestHelper(new LocalCapabilitiesServiceTestConfig());
   private FakeScheduledInterviewDao scheduledInterviewDao;
+  private FakeAvailabilityDao availabilityDao;
   private FakePersonDao personDao;
 
-  @Before
-  public void setUp() {
-    helper.setUp();
-    scheduledInterviewDao = new FakeScheduledInterviewDao();
-    personDao = new FakePersonDao();
-  }
-
-  @After
-  public void tearDown() {
-    helper.tearDown();
-  }
-
-  private final Person interviewer =
+  private final Person person1 =
       Person.create(
```

Comment:
This really is way too many sequential string params, but I guess we can't change that in this PR.

These get really easy to mix up once there's more than like two of them in a row.

Just don't do this in future code you write.